### PR TITLE
OEL-1147 - Footer

### PR DESCRIPTION
**Pull request by escuriola**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -34,7 +34,8 @@
         "openeuropa/oe_starter_content": "1.x-dev",
         "openeuropa/task-runner-drupal-project-symlink": "^1.0",
         "phpspec/prophecy-phpunit": "^1 || ^2",
-        "openeuropa/oe_contact_forms": "~1.1"
+        "openeuropa/oe_contact_forms": "~1.1",
+        "drupal/twig_tweak": "^3.1"
```
Comment (drishu) [inline composer.json]: are we still using twig_tweak ?

**Review by drishu — CHANGES_REQUESTED** [review]

File: modules/oe_whitelabel_helper/oe_whitelabel_helper.info.yml [hunk 2]
```diff
@@ -6,5 +6,7 @@ core_version_requirement: ^8.9 || ^9.1
 
 dependencies:
   - oe_bootstrap_theme:oe_bootstrap_theme_helper
+  - openeuropa:oe_corporate_blocks
+  - drupal:twig_tweak
```
Comment (drishu) [inline modules/oe_whitelabel_helper/oe_whitelabel_helper.info.yml]: remove this

File: tests/src/Kernel/FooterBlockTest.php [hunk 3]
```diff
@@ -17,17 +17,14 @@ class FooterBlockTest extends SparqlKernelTestBase {
...
     'oe_corporate_blocks',
     'oe_corporate_site_info',
     'oe_whitelabel_helper',
     'rdf_skos',
     'system',
-    'ui_patterns',
-    'ui_patterns_library',
-    'ui_patterns_settings',
     'user',
+    'twig_tweak',
```
Comment (drishu) [inline tests/src/Kernel/FooterBlockTest.php]: remove

File: templates/overrides/navigation/oe-corporate-blocks-eu-footer.html.twig [hunk 4]
```diff
@@ -1,110 +1,97 @@
...
-  {# Follow section must always be the last on the left. #}
-  {% set _section_2 = _section_2|merge([{
-    'title': 'Follow us on social media'|t,
-    'links': bcl_footer_links(site_specific_footer.social_links),
-  }]) %}
-{% endif %}
+  {% set eu_logo %}
+    <picture>
+      <source media="(max-width: 992px)"
+              srcset="{{ bcl_logo_path ~ 'eu/mobile/logo-eu--' ~ current_language_id|to_internal_language_id ~ '.svg' }}">
```
Comment (drishu) [inline templates/overrides/navigation/oe-corporate-blocks-eu-footer.html.twig:29]: some rebasing is required, this filter `to_internal_language_id` is missing in this branches code

**Review by drishu — CHANGES_REQUESTED** [review]

**Review by drishu — DISMISSED** [review]

File: templates/overrides/navigation/oe-corporate-blocks-neutral-footer.html.twig [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+        'items': _section_2,
+      },
+      {
+        'classes': 'col-12 col-lg-4',
+        'items': _section_3,
+      }
+    ],
+  } %}
+
+  {% include '@oe-bcl/bcl-footer/bcl-footer.html.twig' with {
```
Comment (brummbar) [inline templates/overrides/navigation/oe-corporate-blocks-neutral-footer.html.twig]:
```suggestion
  {% include '@oe-bcl/footer' with {
```

Comment (brummbar) [reply templates/overrides/navigation/oe-corporate-blocks-neutral-footer.html.twig]: Same in the other 3 templates.

**Review by brummbar — CHANGES_REQUESTED** [review]

**Review by drishu — CHANGES_REQUESTED** [review]
there is a duplicate `div.col-12.col-lg-4`
Screenshot 2022-03-14 at 16 08 21

**Review by drishu — APPROVED** [review]

**Review by abel-santos-corral — APPROVED** [review]

**Review by Maxfire — APPROVED** [review]

**Review by brummbar — APPROVED** [review]

**Pull request merged**